{{ cookiecutter.format }}/app/build.gradle

```diff
@@ -22,8 +22,15 @@ android {
 
         python {
             version "{{ cookiecutter.python_version|py_tag }}"
+
+
             pip {
                 install "-r", "requirements.txt"
+
+                def f = new File(project.projectDir, "pip-options.txt")
+                def pipOptions = f.isFile() ? f.getText("utf-8").split("\n") : []
+                pipOptions = pipOptions.findAll { !it.startsWith("#") && !it.isBlank() }
```

Comment:
Comment support may not be necessary – let's discuss at https://github.com/beeware/briefcase/pull/2059#discussion_r1844721349.